### ISSUE TITLE
Missing Zcash Utils

### ISSUE THREAD

**Issue by fubuloubu** [GunClear/Silencer#8]
Attempted to compile, ran into this issue.

```bash
[ 98%] Building CXX object src/CMakeFiles/main.dir/main.cpp.o
/home/bryant/work/GunClear/Silencer/src/main.cpp:39:10: fatal error: zcash/util.h: No such file or directory
 #include <zcash/util.h>
          ^~~~~~~~~~~~~~
compilation terminated.
src/CMakeFiles/main.dir/build.make:62: recipe for target 'src/CMakeFiles/main.dir/main.cpp.o' failed
make[2]: *** [src/CMakeFiles/main.dir/main.cpp.o] Error 1
CMakeFiles/Makefile2:3238: recipe for target 'src/CMakeFiles/main.dir/all' failed
make[1]: *** [src/CMakeFiles/main.dir/all] Error 2
Makefile:129: recipe for target 'all' failed
make: *** [all] Error 2
```

Is Zcash supposed to be a submodule?

**Comment by SRCoughlin** [comment]
I've been using the utility functions for ease-of-development. I'll refactor and prepare for attribution.

**Comment by SRCoughlin** [comment]
Should be resolved in #62669f3e79cc87421716f8010b318f7e0a1d322c
